src/App.tsx

```diff
@@ -1,9 +1,31 @@
 import './App.scss';
+import {
+  Link, Redirect, Route, Switch,
+} from 'react-router-dom';
+import { HomePage } from './Components/HomePage/HomePage';
+import { PeoplePage } from './Components/PeoplePage/PeoplePage';
+import { NotFoundPage } from './Components/NotFoundPage/NotFoundPage';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App: React.FC = () => {
+  return (
+    <div className="App">
+      <Link to="/" className="nav-link">
+        Home Page
+      </Link>
+
+      <Link to="/peoples" className="nav-link">
+        Peoples
+      </Link>
+
+      <Switch>
+        <Route path="/" exact component={HomePage} />
+        <Route path="/peoples" component={PeoplePage} />
+        <Redirect path="/home" to="/" />
```

Comment:
You use Redirect, but you have never used "/home", so your Redirect is nonsensical. Though conditions of the task don't order it exactly.